src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -104,7 +104,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
             params: {
               screen: 'HomeTabs',
               params: {
-                screen: 'ChannelList',
+                screen: from ?? 'ChannelList',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `screen` parameter of the navigation params object now defaults to the value of `from` if it exists, otherwise it defaults to `'ChannelList'`. This change makes the navigation behavior more dynamic based on the `from` parameter.
```diff
- screen: 'ChannelList',
+ screen: from ?? 'ChannelList',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->